crates/typst-syntax/src/package.rs

```diff
@@ -423,4 +482,34 @@ mod tests {
         assert!(!v1_1_1.matches_lt(&VersionBound::from_str("1.1").unwrap()));
         assert!(v1_1_1.matches_lt(&VersionBound::from_str("1.2").unwrap()));
     }
+
+    #[test]
+    fn minimal_manifest() {
+        assert!(toml::from_str::<PackageManifest>(
+            r#"
+            [package]
+            name = "package"
+            version = "0.1.0"
+            entrypoint = "src/lib.typ"
+        "#,
+        )
+        .is_ok());
```

Comment:
I think if we're gonna have this test, it should validate that at least some part of the manifest actually parses as intended.